zypper.spec.cmake

```diff
@@ -95,9 +95,6 @@ Summary:        aptitude compatibility with zypper
 Group:          System/Packages
 Requires:       perl
 Requires:       zypper
-%if 0%{?suse_version}
-Supplements:    zypper
```

Comment:
Change this to `Supplements: (zypper and perl)` if you only want it pulled in when Perl is already installed.

Alternatively, you could do `Supplements: (zypper unless openSUSE-MicroOS-release)` to prevent it from supplementing on MicroOS systems.

Reply:
hardcoding that logic into packages makes it hard to create derivatives. We have patterns as central place to control what's installed by default instead.